javascript/src/visitor/zh.utils.ts

```diff
@@ -153,16 +153,27 @@ export function parseZhValueWord(ctx: TerminalNode): number | null {
 const nzh = new Nzh({
   ch: "零一二三四五六七八九",
   ch_u: "个十百千万亿兆京",
+  ch_f: "负",                   // 负字符
```

Comment:
Can you add test cases for this?

Reply:
aha, this is copied from Nzh example, I think we haven't have a case of  negative time. So could I replace it to empty char?